tests/integration/snapshots/integration__artichoke__ruby_app__hello_world.rb.snap

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
how do these snaps get updated if there is an intentional change? Do you think we can or should add docs to `CONTRIBUTING.md` for this?

Reply:
Reading my mind. I was/am planning on adding something to `CONTRIBUTING.md`

Just so you know though, all test runs will test whether the output matches. If it doesn't, the test execution fails (nice), but also a draft/review snap is created. Those can be viewed via `cargo insta review` which is just a small CLI that allows for easy review.

They can all be forced to be accepted via a command, but snapshot tests should be done consciously. e.g. I'm accepting this snapshot, because I know it is changing, and I accept said changes. They can almost be seen as a reflection of the execution. This does mean you don't want to test rand functions etc, since the output isn't predictable. They're just a safe guard at the end of the day.